tla/consistency/ExternalHistoryInvars.tla

```diff
@@ -205,7 +205,56 @@ CommittedRwLinearizableInv ==
     /\ AllCommittedObservedInv
     /\ AtMostOnceObservedInv
 
-\*Debugging invariants to check that specific states are reachable
+----
+
+\* Ordering over txIDs, the form of which is <<view,seqnum>>
+TxIDStrictlyLessThan(x, y) ==
+    \/ x[1] < y[1]
```

Comment:
A less invasive refactoring than the introduction of records would be the addition of two definitions such as `View==1` and `SeqNum==2` to be able to write `x[View] < y[View] ...`.

Reply:
Reworked in https://github.com/microsoft/CCF/pull/6185/commits/4123959ee61f5708932a566c54fce11d134a99d1 but I think the way to go is your previous suggestion and replace the `tx_id` into the separate `view` and `seq_num` fields. I'll save that for a future PR